src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/AppointmentDateTime.jsx

```diff
@@ -2,16 +2,18 @@ import React from 'react';
 import {
   getTimezoneAbbrBySystemId,
   getTimezoneDescFromAbbr,
+  getUserTimezoneAbbr,
+  stripDST,
 } from '../../../utils/timezone';
 
-export function getAppointmentTimezoneAbbreviation(facilityId) {
+function getAppointmentTimezoneAbbreviation(facilityId) {
   if (!facilityId) {
-    return null;
+    return stripDST(getUserTimezoneAbbr());
```

Comment:
This is the main change, we want to show the user's timezone if there's no facility id.